tests/playbooks/roles/install-csi-cinder/tasks/main.yaml

```diff
@@ -158,7 +158,6 @@
         -storage.testdriver=tests/e2e/csi/cinder/test-driver.yaml \
         -ginkgo.focus='External.Storage' \
         -ginkgo.skip='\[Disruptive\]|\[Testpattern:\s+Dynamic\s+PV\s+\(default\s+fs\)\]\s+provisioning\s+should\s+mount\s+multiple\s+PV\s+pointing\s+to\s+the\s+same\s+storage\s+on\s+the\s+same\s+node|\[Testpattern:\s+Dynamic\s+PV\s+\(default\s+fs\)\]\s+provisioning\s+should\s+provision\s+storage\s+with\s+any\s+volume\s+data\s+source\s+\[Serial\]' \
-        -ginkgo.no-color \
```

Comment:
For some reason, the ginkgo flags are now `--ginkgo.X`. Does it make sense to change all to the new format?

See: https://dl.k8s.io/v1.25.3/kubernetes-test-linux-amd64.tar.gz
```
kubernetes/test/bin/e2e.test --help
Controlling Test Order
  --ginkgo.seed [int] (default: randomly generated by Ginkgo)
    The seed used to randomize the spec suite.
  --ginkgo.randomize-all
    If set, ginkgo will randomize all specs together.  By default, ginkgo only
    randomizes the top level Describe, Context and When containers.

Controlling Test Parallelism
These are set by the Ginkgo CLI, do not set them manually via go test.
Use ginkgo -p or ginkgo -procs=N instead.
  --ginkgo.parallel.process [int] (default: 1)
    This worker process's (one-indexed) process number.  For running specs in
    parallel.
...
```

Reply:
let me give a try.. thanks